platforms/darwin-arm64/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@ffprobe-installer/darwin-arm64",
   "version": "5.0.1",
   "description": "Mac OS X FFmpeg binary used by ffprobe-installer",
-  "homepage": "https://formulae.brew.sh/formula/ffmpeg",
+  "homepage": "https://github.com/wongyiuhang",
```

Comment:
I appreciate the acknowledgment. However, I was just compiling from the FFmpeg source. Anyone with an Apple Silicon Mac can do that. I prefer to referring "homepage" to [FFmpeg](https://github.com/FFmpeg/FFmpeg) or the [build guide](https://gist.github.com/wongyiuhang/d207aaf554e1985dd283a2fa1df949af) instead.

Reply:
Ok, I've changed the link to your build guide gist. Yes "anyone can do it" but easier for someone with knowledge/experience, you saved the rest of us a few minutes at least so thanks! 🙏